reconcile/utils/ocm.py

```diff
@@ -948,6 +950,99 @@ def create_github_idp_teams(self, spec):
         }
         self._post(api, payload)
 
+    def _check_oidc_idp_params(self, oidc_idp: OCMOidcIdp, attrs: Iterable[str]):
```

Comment:
Isn't that something the OCMOidcIdp should provide?

Reply:
Unfortunately not, the attributes must be optional, but you have to specify at least one of them. The `OCMOidcIdp` is a `Protocol` and I can't implement a nice pydantic validator for it.